python/GafferArnoldTest/ArnoldRenderTest.py

```diff
@@ -1478,5 +1478,71 @@ def testCoordinateSystem( self ) :
 			# node to have been created for ours.
 			self.assertIsNone( arnold.AiNodeLookUpByName( universe, "/coordinateSystem" ) )
 
+	@GafferTest.TestRunner.PerformanceTestMethod( repeat = 1 )
+	def testInstancerPerf( self ) :
+
+		s = Gaffer.ScriptNode()
+
+		s["plane"] = GafferScene.Plane()
+		s["plane"]["divisions"].setValue( imath.V2i( 500 ) )
+
+		s["sphere"] = GafferScene.Sphere()
+		s["sphere"]["expression"] = Gaffer.Expression()
+
+		s["pathFilter"] = GafferScene.PathFilter()
+		s["pathFilter"]["paths"].setValue( IECore.StringVectorData( [ '/plane' ] ) )
+
+		s["instancer"] = GafferScene.Instancer()
+		s["instancer"]["in"].setInput( s["plane"]["out"] )
+		s["instancer"]["filter"].setInput( s["pathFilter"]["out"] )
+		s["instancer"]["prototypes"].setInput( s["sphere"]["out"] )
+
+		s["options"] = GafferScene.CustomOptions()
+		s["options"]["in"].setInput( s["instancer"]["out"] )
+
+		s["render"] = GafferArnold.ArnoldRender()
+		s["render"]["in"].setInput( s["options"]["out"] )
+		s["render"]["mode"].setValue( s["render"].Mode.SceneDescriptionMode )
+		s["render"]["fileName"].setValue( self.temporaryDirectory() / "test.ass" )
+
+		with Gaffer.Context() as c :
+			c["scene:render:sceneTranslationOnly"] = IECore.BoolData( True )
+			with GafferTest.TestRunner.PerformanceScope() :
+				s["render"]["task"].execute()
+
+	@GafferTest.TestRunner.PerformanceTestMethod( repeat = 1 )
+	def testInstancerEncapsulatePerf( self ) :
+
+		s = Gaffer.ScriptNode()
+
+		s["plane"] = GafferScene.Plane()
+		s["plane"]["divisions"].setValue( imath.V2i( 2000 ) )
```

Comment:
I can see some value in keeping it the same as testInstancePerf for comparison, and I'll go ahead and do that for now. To explain my process though - I also usually aim for 1-2s, though I had tuned this for when we did have the Arnold special case, in which case this test was testing something fairly different, since the Arnold instancer didn't seem to expand during a sceneTranslationOnly render - so it was more just testing that we took minimal time on our end, and wasn't really testing the same thing.

Having abandoned the Arnold specific case now, this is actually expanding properly, so it is comparable. It was only taking 3s for me though, so just slightly over a 1-2s target. Having now matched the settings to testInstancePerf for comparison, it's only taking 0.21s for me, which is probably a bit lower than ideal for getting accurate results.

It is rather concerning that your numbers look so different from mine, but I'll discuss that more below.